.github/.dependabot.yml

```diff
@@ -1,14 +0,0 @@
-# Please see the documentation for all configuration options:
```

Comment:
The security updates will be handled on our side with a tool we have - no more need for dependabot PRs